pipelines/rj_ssm__celular_seguro/flow.py

```
@@ -0,0 +1,51 @@
+
+    headers = {
+        "Content-Type": "application/json"
+    }
+
+
+    response = requests.post(
+        url,
+        json=payload,
+        headers=headers,
+        cert=(crt_path, key_path),
+        verify=False,
```

Ruff (0.15.21)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)